sheets/1/exercise-3.tex

```diff
@@ -57,32 +75,39 @@
     \item \textbf{[1]} Eine Anweisung endet immer mit einem ; und muss im Codeblock, einer Operation, stehen.
     \item \textbf{[2]} Das erzeugen eines Objekts wird mit dem Schlüsselwort new eingeführt. Darauffolgt der Name der Klasse und (). Manchmal werden in die () noch Argumente geschreiben, mit Dingen die das Objekt von Anfang an braucht.
           \begin{lstlisting}[title=\textbf{Konstruktor Syntax},firstnumber=5]
-    new <Typ Name>();
-    new <Typ Name>(<Wert>);
-    new <Typ Name>(<Wert>,<Wert>);
+    new <Typ Name>();// Ein Objekt wird erzeugt
+    new <Typ Name>(<Wert>); //Für das erstellen wird ein Wert mit gegeben
+    new <Typ Name>(<Wert>,<Wert>); //hier sind es 2
     \end{lstlisting}
           \begin{lstlisting}[title=\textbf{Beispiel (1/3) Konstruktor},firstnumber=2,frame=lr]
         new PDFCreator("jvk title",pdfCollector);
     \end{lstlisting}
-    \item \textbf{[2]} Wenn wir ein Objekt länger als nur einmal brauchen müssen wir uns dieses Merken. Dazu verwenden wir eine Variable. Um eine zu erstellen: schreiben wir zuerst die Klasse zu der das Objekt gehört, gefolgt von dem frei wälbaren Namen, den die Variable haben soll. Mit einem \lstinline{=} weisen wir ein Objekt einer Variable zu, indem wir entweder das \lstinline{new Objekt()} direkt dahinter schreiben oder eine andere
+    \item \textbf{[3]} Wenn wir ein Objekt länger als nur einmal brauchen müssen wir uns dieses Merken. Dazu verwenden wir eine Variable. Um eine zu erstellen: schreiben wir zuerst die Klasse zu der das Objekt gehört, gefolgt von dem frei wälbaren Namen, den die Variable haben soll. Mit einem \lstinline{=} weisen wir ein Objekt einer Variable zu, indem wir entweder das \lstinline{new Objekt()} direkt dahinter schreiben oder eine andere
           Variable die bereits einen Wert hat.
           Wenn wir ein Objekt einem Wert zugewiesen haben, können wir es unter dem Namen der Variable wiederfinden.
           \begin{lstlisting}[title=\textbf{Variablen Syntax}]
-    <Typ> <Name>;
-    <Name> = <Klassen Name>();
-    <Name> = <Klassen Name(<Wert>);
-    <Typ> <Name> = <Klassen Name>();
+    <Typ> <Name>; // Variable wird zur späteren Verwendung erzeugt
+    <Name> = new <Klassen Name>(); //Ein Objekt wird erstellt und einer bereits existierenden Variable zugeweisen.
```

Comment:
Wenn die Zeilen zu lang werden so kann man auch folgendes machen:

```
// kommentar
example

// kommentar...
```

So findet man das eh häufiger in java code